### PR TITLE
Final touches

### PR DESCRIPTION
**Pull request by ichsansandy**
- Add sweetalert2 for better dynamic response
- Add number formatter

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+.styled-table td {
+  padding: 15px 25px;
+}
+
+.styled-table tbody tr {
+  border-bottom: 1px solid #ddd;
+}
+
+.styled-table tbody tr:nth-of-type(even) {
+  /* background-color: #f3f3f3; */
```
Comment (Ibizugbe) [inline src/style.css]: Kindly take away commented blocks or pieces of codes from your codebase as they make it messy and give it a less professional look😉.

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+}
+
+.flex-column {
+  display: flex;
+  flex-direction: column;
+}
+
+.space-between {
+  justify-content: space-between;
+}
```
Comment (Ibizugbe) [inline src/style.css]: Nice job so far👍👍. You have an excellent user interface😉. but I think we can make it better by sticking to how the [wireframe](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/images/leaderboard_wireframe.png) was structured😉. 

 See the wire Wireframe
image

From the above, you can see all the contents are fitted to the screen😉 Your input fields look great but I want you to take note of the scores section😉 The width of your score section seems too large and occupies almost all the width of the screen which causes the page to overflow, leading to a poor user experneice🥲. Please let's kindly fix that. Try to make sure the score section takes about 50% of the screen width as shown in the wireframe👍.

image



https://github.com/ichsansandy/leaderboard-microverse/assets/64643995/ffa31084-243a-45a4-b52c-af4d1be5dd54

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi @ichsansandy 👋👋, 

tryagain


Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights✅✅✅

- [x] Gitflow is used👍.
- [x] Linters are passing👍.
- [x] UI looks great👍.


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by osamaashraf6 — APPROVED** [review]
Hi @ichsansandy  😊, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
MinionsGIF (2)

## Highlights🎉 🎉 

- [ ] Nice design😊👌
- [ ] Good Sticking to the wire frame💯
- [ ] Nice styling using the box-shadow to the edges🚀

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**